charts/supermq/values.yaml

```
@@ -279,76 +160,123 @@ postgresqlauth:
         ports:
           postgresql: *postgresqlAuthPort
 
-users:
+certs:
+  enabled: true
   image:
     {}
     # pullSecrets: {}
-    # repository: "magistrala/users"
+    # repository: "supermq/certs"
     # tag: "latest"
     # pullPolicy: "IfNotPresent"
   # jaegerTraceRatio: 1.0
+  # logLevel: error
   # sendTelemetry: true
-  # logLevel: "info"
-  httpPort: 9002
-  grpcPort: 7001
-  adminEmail: "admin@example.com"
-  adminPassword: "12345678"
-  secretKey: "secretKey"
-  passwordRegex: "^.{8,}$"
-  tokenResetEndpoint: "/reset-request"
-  allowSelfRegister: true
-  deleteInterval: "24h"
-  deleteAfter: "720h"
-  # nodeSelector: {}
-  # affinity: {}
-  # tolerations: {}
+  httpPort: 9019
+  logLevel: "error"
+  signCAPath: "/etc/ssl/certs/ca.crt"
+  signCAKeyPath: "/etc/ssl/certs/ca.key"
+  sdkHost: "http://supermq-am-certs"
+  sdkCertsUrl: "${SMQ_CERTS_SDK_HOST}:9010"
+  sdkTlsVerification: "false"
+  vault:
+    url: "http://supermq-vault:8200"
+    approleRoleid: supermq
+    approleSecret: supermq
+    namespace: supermq
+    thingsCertsPkiPath: pki_int
+    thingsCertsPkiRoleName: supermq_things_certs
 
-postgresqlusers:
-  ## If you want to use an external database, set this to false and change host & port
+postgresqlcerts:
+  ## If you want to use an external database, set this to false and change host & port to external postgresql server host & port respectively
   enabled: true
-  name: postgresql-users
-  host: postgresql-users
-  port: &postgresqlUsersPort 5432
-  database: &postgresqlUsersDatabase users
-  username: &postgresqlUsersUsername magistrala
-  password: &postgresqlUsersPassword magistrala
+  name: postgresql-certs
+  host: postgresql-certs
+  port: &postgresqlCertsPort 5432
+  database: &postgresqlCertsDatabase certs
+  username: &postgresqlCertsUsername supermq
+  password: &postgresqlCertsPassword supermq
   global:
     postgresql:
       auth:
-        postgresPassword: *postgresqlUsersPassword
-        username: *postgresqlUsersUsername
-        password: *postgresqlUsersPassword
-        database: *postgresqlUsersDatabase
+        postgresPassword: *postgresqlCertsPassword
+        username: *postgresqlCertsUsername
+        password: *postgresqlCertsPassword
+        database: *postgresqlCertsDatabase
       service:
         ports:
-          postgresql: *postgresqlUsersPort
+          postgresql: *postgresqlCertsPort
```

Please add resource limitation and presistance size for all postgres. 
If we don't provide this constraint, the in local machine k8s it is not running, All postgres using full memory 

Please analyze further and try to reduce the resource. 
```suggestion
  resources:
    requests:
      cpu: 25m
  persistence:
    size: 500Mi
```